src/pid/issuing.ts

```diff
@@ -302,4 +304,39 @@ export class Issuing {
 
     throw new PidIssuingError(`Unable to obtain credential!`);
   }
+
+  /**
+   * Obtain the PID issuer metadata
+   *
+   * @function
+   * @returns PID issuer metadata
+   *
+   */
+  async getMetadata(): Promise<PidIssuerMetadata> {
```

Comment:
**suggestion(naming):** Shouldn't it be `getEntityConfiguration`?‏

Reply:
yep! fixed!